dll/cpl/access/general.c

```diff
@@ -194,8 +195,10 @@ GeneralPageProc(HWND hwndDlg,
                            IDC_RESET_BOX,
                            pGlobalData->accessTimeout.dwFlags & ATF_TIMEOUTON ? BST_CHECKED : BST_UNCHECKED);
             FillResetComboBox(GetDlgItem(hwndDlg, IDC_RESET_COMBO));
-            SendDlgItemMessage(hwndDlg, IDC_RESET_COMBO, CB_SETCURSEL,
-                               (pGlobalData->accessTimeout.iTimeOutMSec / 300000) - 1, 0);
+#define FIVE_MINS_IN_MS (5 * 60 * 1000)
```

Comment:
Move this define to the start of the file.
And update old line 236 too.

Reply:
Fixed.